includes/forum-database.php

```diff
@@ -170,6 +170,8 @@ public function buildDatabase() {
             post_id int(11) NOT NULL default '0',
             user_id int(11) NOT NULL default '0',
             reaction varchar(20) NOT NULL default '',
+			author int(11) NOT NULL default '0',
```

Comment:
Better to use `author_id` instead of `author` as a variable-name to make it more clear that we have an ID here.

Reply:
I agree!